app/Filament/Resources/RosterUpdateResource.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace App\Filament\Resources;
+
+use App\Filament\Resources\RosterUpdates\Pages\ListRosterUpdates;
+use App\Filament\Resources\RosterUpdates\Pages\ViewRosterUpdate;
+use App\Models\RosterUpdate;
+use App\Models\Training\WaitingList;
+use Filament\Forms\Components\DatePicker;
+use Filament\Forms\Components\Section;
+use Filament\Forms\Components\TextInput;
+use Filament\Forms\Form;
+use Filament\Resources\Resource;
+use Filament\Tables\Actions\ViewAction;
+use Filament\Tables\Columns\TextColumn;
+use Filament\Tables\Table;
+
+class RosterUpdateResource extends Resource
```

Comment:
Do you need any permissions behind this?

Reply:
i.e. a policy etc.. Could use the existing roster.manage permission?

Reply:
Good point - added in 91a1016